lib/cocoapods/target/pod_target.rb

```diff
@@ -465,6 +484,16 @@ def app_target_label(app_spec)
       "#{label}-#{subspec_label(app_spec)}"
     end
 
+    def app_host_target_label(test_spec)
```

Comment:
docs, open to a better name for this method

Reply:
Worth splitting into two functions?